bang/magicollections.py

```diff
@@ -286,18 +288,25 @@ def share_image(self, context, item):
     def to_fields(self, view, item, exclude_fields=None, *args, **kwargs):
         if exclude_fields is None: exclude_fields = []
         exclude_fields.append('d_names')
+        if item.school is not None:
+            exclude_fields.append('classroom')
+        exclude_fields.append('japanese_name')
         fields = super(MemberCollection, self).to_fields(view, item, *args, icons=MEMBERS_ICONS, images={
             'astrological_sign': '{}img/i_astrological_sign/{}.png'.format(RAW_CONTEXT['static_url'], item.i_astrological_sign),
         }, exclude_fields=exclude_fields, **kwargs)
         if 'square_image' in fields:
             del(fields['square_image'])
+        if item.classroom is not None and item.school is not None:
+            setSubField(fields, 'school', key='type', value='html')
+            setSubField(fields, 'school', key='value', value= mark_safe(u'<b>{} <span class="text-muted">({})</span></b>'.format(item.school, item.classroom)))
```

Comment:
You don't need `mark_safe` if `type` is `html`.

Reply:
✅